### PR TITLE
Cron: fix schedule_corpus_pruning to work for CUSTOM_BINARY jobs too.

### PR DESCRIPTION
**Pull request by Dor1s**
Bug: https://bugs.chromium.org/p/chromium/issues/detail?id=963221

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
Maybe add a test to ensure this too when/if you get a chance.

**Comment by Dor1s** [comment]
Please review the tests then :)

File: src/appengine/handlers/cron/schedule_corpus_pruning.py [hunk 1]
```diff
@@ -41,24 +41,37 @@ def _get_latest_job_revision(job):
...
+
+    if utils.string_is_true(job.get_environment().get('CUSTOM_BINARY')):
+      # Custom binary jobs do not have revisions.
+      latest_revision = None
+    else:
+      latest_revision = _get_latest_job_revision(job)
+      if not latest_revision:
+        continue
+
+    queue_name = tasks.queue_for_job(job.name)
```
Comment (jonathanmetzman) [inline src/appengine/handlers/cron/schedule_corpus_pruning.py]: Why change the variable name from queue to queue_name?

Comment (Dor1s) [reply src/appengine/handlers/cron/schedule_corpus_pruning.py]: Found myself confused by `queue`, what type that variable was. Don't have strong opinion here though. Also unsure about `tasks_to_schedule`, maybe `tasks_arguments` or `parameters` or just `result` :)

File: src/appengine/handlers/cron/schedule_corpus_pruning.py [hunk 2]
```diff
@@ -41,24 +41,37 @@ def _get_latest_job_revision(job):
...
+    for target_job in fuzz_target_utils.get_fuzz_target_jobs(job=job.name):
+      task_target = target_job.fuzz_target_name
+      if latest_revision:
+        task_target += '@%s' % latest_revision
+
+      tasks_to_schedule.append((task_target, job.name, queue_name))
```
Comment (oliverchang) [inline src/appengine/handlers/cron/schedule_corpus_pruning.py]: nit: no need for the list and appends. just use `yield`

**Pull request merged**